### PR TITLE
Roy branch17 dec16

### PR DESCRIPTION
**Pull request by royru**

### CODE REVIEW

File: Chat/public/user.js [hunk 1]
```diff
@@ -208,17 +208,13 @@ var ChatControls = (function($) {
 
 // #chat-wrapper
 var MessageWindow = (function($) {
-
-    var appendGeneric = function(generic) {
-        $('#content').append("<div class=\"message-wrapper\"><div style=\"background-color: yellow\" class=\"message\">da "+
-            "chunt was here wemmer es feld usfüllt, muesmer no anderst gstalte: you filled out the generic message "+
-            "[" + generic.c.id + "] with value " + generic.c.v + "</div></div>");   
-    }
```
Comment (Philippw8) [inline Chat/public/user.js]: why was that removed?

Comment (royru) [inline Chat/public/user.js]: Replaced with another function in this module

File: Chat/socketHandler.js [hunk 2]
```diff
@@ -623,7 +644,7 @@ SocketHandler.prototype.start = function(server) {
                             sendMessageToProfileSockets(sockets, data);
                         }));
 
-                        fn(true);
+                        setTimeout(() => {fn(true)}, 1000);
```
Comment (Philippw8) [inline Chat/socketHandler.js]: why do we need a timeout? @royru

Comment (royru) [inline Chat/socketHandler.js]: aah, this is not really needed of course, just to show something related to
acks

2015-12-20 10:14 GMT+01:00 Philippw8 <notifications@github.com>:
